### PR TITLE
Feature create flyout

### PR DESCRIPTION
**Pull request by kwood15**
- Have re-structured the flyout files as discussed.
- Basic styles applied to flyout - toggle left as it is for now.
- Have seen a couple of minor syntax errors on the card component files so have fixed in this branch so all checked.

image

### CODE REVIEW

File: demo/src/Components/Cards/index.jsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ const Cards = () => (
       <h4>Cards— Base</h4>
       <div className="o-grid o-grid--gutter-md o-grid--matrix">
 
-        <div className="o-grid__item u-1/3">
+        <div className="o-grid__item u-1/3@sm">
```
Comment (rbrtsmith) [inline demo/src/Components/Cards/index.jsx:9]: Good catch :+1:

File: nebula-components/settings/components/_flyout.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+$nb-flyout-min-width: 220px !default;
+$nb-flyout-background: $nb-color-white !default;
```
Comment (rbrtsmith) [inline nebula-components/settings/components/_flyout.scss]: We can remove these redundant variables now.

File: nebula-components/tools/components/_flyout.scss [hunk 3]
```diff
@@ -0,0 +1,33 @@
+@mixin nb-triangle($direction, $size: $nb-flyout-triangle-size, $color: $nb-flyout-triangle-color) {
```
Comment (rbrtsmith) [inline nebula-components/tools/components/_flyout.scss]: We should move this file into `tools/` and call it `_triangle.scss` as it's a global mixin.

**Review by rbrtsmith — CHANGES_REQUESTED** [review]
Really good work! just a few small tweaks that I commented on then this will be good to be merged in…

File: nebula-components/settings/components/_flyout.scss (hunk 2 above)
Comment (kwood15) [reply nebula-components/settings/components/_flyout.scss]: Done

File: nebula-components/tools/components/_flyout.scss (hunk 3 above)
Comment (kwood15) [reply nebula-components/tools/components/_flyout.scss]: Done

**Comment by kwood15** [comment]
All done

**Review by rbrtsmith — APPROVED** [review]

**Pull request merged**